helix-term/src/main.rs

```diff
@@ -85,12 +85,13 @@ FLAGS:
         std::process::exit(0);
     }
 
-    let conf_dir = helix_core::config_dir();
-    if !conf_dir.exists() {
-        std::fs::create_dir_all(&conf_dir).ok();
+    let config_dir = helix_core::config_dir();
+    if !config_dir.exists() {
+        std::fs::create_dir_all(&config_dir).ok();
     }
 
-    let config = match std::fs::read_to_string(conf_dir.join("config.toml")) {
+    // `config.toml`
+    let config = match std::fs::read_to_string(config_dir.join("config.toml")) {
```

Comment:
Why not move this into application.rs now?

Reply:
I wasn't sure if it would interfere with the test harness you're working on.